src/plays/simple-live-chat/components/chat/chat-footer.scss

```diff
@@ -0,0 +1,55 @@
+@import "../../variables";
+
+.simple-live-chat-chat-footer {
+  position: sticky;
+  bottom: -10px;
+}
+
+.simple-live-chat-footer-container {
+  display: flex;
+  padding-bottom: 0;
+  justify-content: center;
+  width: 600px;
+
+  @media (max-width: 640px) {
+    width: 100%;
+  }
+
+  textarea {
```

Comment:
Better not to use element selector, it might interfere with the other even though it resides in leave level component.

Reply:
yes a className is added